help/C/Help_ch_Transactions.xml

```diff
@@ -88,12 +97,191 @@ Translators:
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <figure>
+          <title>Example Register in <guimenuitem>Auto-Split Ledger</guimenuitem>,
+            <guimenuitem>Double Line</guimenuitem> view
+          </title>
+          <screenshot id="reg-views-ViewAutoSplitDouble">
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/Help_Tx_ViewAutoSplitDouble.png"
+                  format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+              </imageobject>
+              <imageobject role="fo">
+                <imagedata fileref="figures/Help_Tx_ViewAutoSplitDouble.png" format="PNG"
+                           srccredit="Chris Good"></imagedata>
+              </imageobject>
+
+              <textobject>
+                <phrase>Example Clothes register</phrase>
+              </textobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <figure>
+          <title>Example Register in <guimenuitem>Transaction Journal</guimenuitem>,
+            <guimenuitem>Single Line</guimenuitem> view
+          </title>
+          <screenshot id="reg-views-ViewTxJnl">
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/Help_Tx_ViewTxJnl.png"
+                  format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+              </imageobject>
+              <imageobject role="fo">
+                <imagedata fileref="figures/Help_Tx_ViewTxJnl.png" format="PNG"
+                           srccredit="Chris Good"></imagedata>
+              </imageobject>
+
+              <textobject>
+                <phrase>Example Clothes register</phrase>
+              </textobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <figure>
+          <title>Example Register in <guimenuitem>Transaction Journal</guimenuitem>,
+            <guimenuitem>Double Line</guimenuitem> view
+          </title>
+          <screenshot id="reg-views-ViewTxJnlDouble">
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/Help_Tx_ViewTxJnlDouble.png"
+                  format="PNG" srccredit="Chris Good" width="510px"></imagedata>
+              </imageobject>
+              <imageobject role="fo">
+                <imagedata fileref="figures/Help_Tx_ViewTxJnlDouble.png" format="PNG"
+                           srccredit="Chris Good"></imagedata>
+              </imageobject>
+
+              <textobject>
+                <phrase>Example Clothes register</phrase>
+              </textobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+      </sect3>
+    </sect2>
 
+    <sect2 id="reg-views-filter-sort">
+      <title>Filter or Sort</title>
+      <para>The two other options to change the view of the register are; to restrict displayed items with the
```

Comment:
semi-colon grammatically incorrect here. Should be colon. Also, stylistically might prefer prefixing each option with a number.